src/driver/amdxdna/aie2_ctx.c

```diff
@@ -216,13 +218,15 @@ void aie2_hwctx_resume(struct amdxdna_hwctx *hwctx)
 			  hwctx->name, hwctx->status, err);
 }
 
-static inline void
+static void
 aie2_sched_notify(struct amdxdna_sched_job *job)
 {
+	struct amdxdna_hwctx *hwctx = job->hwctx;
 	struct dma_fence *fence = job->fence;
 
-	job->hwctx->completed++;
-	trace_xdna_job(&job->base, job->hwctx->name, "signaling fence", job->seq);
+	amdxdna_update_stats(hwctx->client, ktime_get(), false);
```

Comment:
Based on perf numbers need to see if this needs to be guarded with some debug macro? 
otherwise LGTM.

Reply:
define AMDXDNA_DRM_USAGE to control this feature